app/date-comment-helpers.js

```diff
@@ -34,50 +34,43 @@ class DateCommentHelpers {
     if (correctComment) {
       const commentDateMatch = myRegex.exec(correctComment.data.text);
       const commentDate = commentDateMatch[1];
-      const commMoment = moment(commentDate, 'MM/DD/YYYY').toISOString();
-      return commMoment;
+      const isoDate = moment(commentDate, 'MM/DD/YYYY').toISOString();
+      return isoDate;
     }
     return false;
   }
 
-  findPrevMoveDateFromComments(opts) {
-    const actionList = opts.actionList;
-    const cardCreationDate = opts.cardCreationDate;
-    const commentList = opts.commentList;
-    let correctComment = false;
-    if (commentList) {
-      correctComment = this.checkCommentsForDates(commentList, true, false);
-    }
-    if (correctComment) {
-      return correctComment;
-    } else if (actionList) {
-      if (actionList.length > 1) {
-        return actionList[0].date;
-      }
+  getNewCommentFromDate(deletedNewComment, comments, altToDate) {
+    if (altToDate) {
+      return moment(this.checkCommentsForDates(comments, true, true));
     }
+    const useLastFromDate = !deletedNewComment;
+    return moment(this.checkCommentsForDates(comments, true, useLastFromDate));
+  }
 
-    if (opts.cardCreationDate) {
-      return cardCreationDate;
+  differentToDate(comments, currentTime) {
+    const mostRecentToDate = this.checkCommentsForDates(comments, true, true);
+    const recentToMoment = moment(mostRecentToDate);
+    const differenceFromLastComment = currentTime.diff(recentToMoment, 'days');
+    if (differenceFromLastComment > 2) {
+      return recentToMoment;
     }
-
-    return moment('01/01/2016', 'MM/DD/YYYY').toISOString();
+    return false; // Default to returning today
```

Comment:
Should this be returning a date?  The comment seems to imply that but I'm not sure.